### PR TITLE
Adds Sigfox sensor docs

### PR DESCRIPTION
**Pull request by robmarkcole**
**Description:**
Adds docs for the new Sigfox sensor.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant/pull/13731) (if applicable):** home-assistant/home-assistant#13731

## Checklist:
- [x] The documentation follow the [standards][standards].

### CODE REVIEW

File: source/_components/sensor.sigfox.markdown [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: "Sigfox Sensor"
+description: "Display messages from Sigfox devices in Home Assistant."
+date: 2018-04-07 12:10
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: file.png
```
Comment (frenck) [inline source/_components/sensor.sigfox.markdown]: Incorrect logo. Maybe add the Sigfox logo to this PR?

File: source/_components/sensor.sigfox.markdown [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ha_category: Sensor
+ha_iot_class: "Local Polling"
+ha_release: 0.68
+---
+
```
Comment (frenck) [inline source/_components/sensor.sigfox.markdown:15]: One empty line to many ;)

File: source/_components/sensor.sigfox.markdown [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: sigfox
+    api_login: your_api_login
+    api_password: your_api_password
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/sensor.sigfox.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/sensor.sigfox.markdown [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+sharing: true
+footer: true
+logo: file.png
+ha_category: Sensor
+ha_iot_class: "Local Polling"
+ha_release: 0.68
+---
+
+
+[SigFox](https://www.sigfox.com/en) component adding a sensor for each Sigfox device registered with your account. The default name of sensors is `sigfox_{DEVICE_ID}` where `DEVICE_ID` is the devices Sigfox ID. The state of an added Sigfox sensor is the payload of the last message published by that device. Additionally there are attributes for the latitude and longitude coordinates of the device, as well as the signal-to-noise ratio ( [snr](https://en.wikipedia.org/wiki/Signal-to-noise_ratio)).
```
Comment (frenck) [inline source/_components/sensor.sigfox.markdown]: Missing comma: `Additionally, there are attributes`

**Review by frenck — CHANGES_REQUESTED** [review]
@robmarkcole Thank you for this!
I've reviewed your PR and left some comments. Could you please take a look? 👍

File: source/_components/sensor.sigfox.markdown (hunk 1 above)
Comment (robmarkcole) [reply source/_components/sensor.sigfox.markdown]: Could do, but does HA have a policy on seeking permission from the brand before using their logo?

File: source/_components/sensor.sigfox.markdown (hunk 2 above)
Comment (robmarkcole) [reply source/_components/sensor.sigfox.markdown:15]: ok

File: source/_components/sensor.sigfox.markdown (hunk 4 above)
Comment (robmarkcole) [reply source/_components/sensor.sigfox.markdown]: ok

File: source/_components/sensor.sigfox.markdown (hunk 3 above)
Comment (robmarkcole) [reply source/_components/sensor.sigfox.markdown]: ok

File: source/_components/sensor.sigfox.markdown (hunk 1 above)
Comment (robmarkcole) [reply source/_components/sensor.sigfox.markdown]: https://logos-download.com/11226-sigfox-logo-download.html

Comment (robmarkcole) [reply source/_components/sensor.sigfox.markdown]: OK I got permission from Sigfox to use one of their logos, will add tomorrow

**Comment by robmarkcole** [comment]
@frenck ready for another review :)

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Looks good to me @robmarkcole 👍 
Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
Parent PR has been merged.
I'll go ahead and merge this one in as well 🎉

**Pull request merged**